packages/grid/_modules_/grid/models/params/gridResizeParams.ts

```diff
@@ -0,0 +1,15 @@
+import { GridScrollBarState, GridViewportSizeState } from '../gridContainerProps';
+
+/**
+ * Object passed as parameter onto the resize event handler.
+ */
+export interface GridResizeParams {
+  /**
+   * The viewport sizes state.
+   */
+  viewportSizes: GridViewportSizeState;
```

Comment:
don't we need the size of the full component? 🤔

Reply:
I can ass in the `containerSizes` as well or in favour of this one.

Reply:
I think it would be nice to add just the total size of the component. So users can resize their container...

Reply:
I don't think they would care too much and if so they can get the state using the api

Reply:
Done